homeassistant/components/integration/sensor.py

```diff
@@ -242,6 +303,12 @@ def __init__(
         self._source_entity: str = source_entity
         self._last_valid_state: Decimal | None = None
         self._attr_device_info = device_info
+        self._max_dt: timedelta | None = (
```

Comment:
Bit in doubt on how hard to break with the current behavior. If current users don't update their config, this new config will default to `None` and they don't get time based integration, changes are minimal. 

We could also decide to pick the default of 60s here. Leaving less legacy. But that would trigger sudden state changes of this sensor even if the source doesn't change (which is also the point of this PR).

Reply:
We should always avoid breaking changes

Reply:
I can leave it as is, then current users will get a `None` value for `max_dt` and not get time based integration.